examples/pump-controller-app/nrfconnect/README.md

```diff
@@ -66,15 +66,15 @@ LE advertising manually after powering up the device by pressing **Button 4**.
 
 ### Bluetooth LE rendezvous
 
-In this example, the commissioning procedure is done over Bluetooth LE between a
-CHIP device and the CHIP controller, where the controller has the commissioner
-role.
+In this example, the commissioning procedure (called rendezvous) is done over
```

Comment:
I do see that both READMEs were copied from the Lock application 1 to 1 back then in the initial PR. Could we use this PR to update the title and short description at the top of the documentation? Same with the pump-app?

Reply:
You are right. There is in general too many notions from the lock app. We will create a separate PR with updates to the two pump examples and their documentation to avoid cluttering this one.